src/main/java/sirius/biz/tenants/TenantData.java

```diff
@@ -218,6 +218,11 @@
     public static final Mapping JOURNAL = Mapping.named("journal");
     private final JournalData journal;
 
+    public static final Mapping LANG = Mapping.named("lang");
+    @NullAllowed
```

Comment:
alternative we could set the `lang` in `@BeforeSave` to the `NLS.getDefaultLanguage()` if empty